test/rws_rapid_test.cpp

```diff
@@ -0,0 +1,99 @@
+#include <gtest/gtest.h>
+
+#include "rws_rapid_test.h"
+
+namespace abb :: rws
+{
+
+    TEST(RAPIDArrayTest, testSimpleArrayStringConstruction)
+    {
+        RAPIDArray<MockRAPIDAtomic> test_array {
+            "mock_data",
+            std::vector<MockRAPIDAtomic> {
+                MockRAPIDAtomic {"data_1"},
+                MockRAPIDAtomic {"data_2"},
+                MockRAPIDAtomic {"data_3"}
+            }
+        };
+
+        std::string expected_constructed_string = "[data_1,data_2,data_3]";
+
+        EXPECT_EQ(test_array.constructString(), expected_constructed_string);
+    }
+
+    TEST(RAPIDArrayTest, testDoubleArrayStringConstruction)
```

Comment:
Maybe `NestedArray` is more appropriate, because `DoubleArray` can be interpreted as array of doubles.